app/scripts/background.js

```diff
@@ -1059,6 +1069,32 @@ export function setupController(
     });
   };
 
+  connectRemoteCaip = async (remotePort) => {
```

Comment:
can we just add a comment explaining that this is for non externally_connectable multichain API connections (i.e. Firefox site connections). Perhaps also just noting that the connectExternalCaip is used for chrome etc...?

Reply:
does this rename help? 

https://github.com/MetaMask/metamask-extension/pull/30142/commits/e32e8cf97e117ffe13a7791864d55d2922c45fbc